genie-web/src/main/java/com/netflix/genie/web/services/loadbalancers/script/ScriptLoadBalancer.java

```diff
@@ -245,22 +244,22 @@ public void refresh() {
                     "Invalid empty value for script source file property: " + SCRIPT_FILE_SOURCE_PROPERTY_KEY
                 );
             }
-            final String scriptFileSource = new URI(scriptFileSourceValue).getPath();
+            final String scriptFileSource = new URI(scriptFileSourceValue).toString();
 
             final String scriptFileDestinationValue =
                 this.environment.getProperty(SCRIPT_FILE_DESTINATION_PROPERTY_KEY);
-            if (StringUtils.isBlank(scriptFileSourceValue)) {
+            if (StringUtils.isBlank(scriptFileDestinationValue)) {
                 throw new IllegalStateException(
                     "Invalid empty value for script destination directory property: "
                         + SCRIPT_FILE_DESTINATION_PROPERTY_KEY
                 );
             }
-            final String scriptDestinationDir = new URI(scriptFileDestinationValue).getPath();
+            final String scriptDestinationDir = new URI(scriptFileDestinationValue).toString();
```

Comment:
I think`toString()` here will return `file:///foo/bar/`.
At line 260, this is parsed as `File`. Will the `file://` part get dropped?
Will it fail if someone for some reason tries to pass `s3://...`?